src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsManagedLanguagesProjectSystemServices.cs

```diff
@@ -58,10 +58,11 @@ internal class VsManagedLanguagesProjectSystemServices
 
         static VsManagedLanguagesProjectSystemServices()
         {
-            _referenceMetadata = Array.CreateInstance(typeof(string), 3);
+            _referenceMetadata = Array.CreateInstance(typeof(string), 4);
```

Comment:
Possibly. But we should figure out the reason why was this used in a separate check in.